apps/site/src/content/blog/hello-world.mdx

```diff
@@ -1,49 +1,38 @@
+## What's on the Astrolicious Horizon? 🔭
 
-    ## What's on the Astrolicious Horizon? 🔭
+Beneath our banner, stars are already aligning - expect a universe of projects, guides, and docs to explore soon. Have a look at who joined our galaxy already:
 
-    Beneath our banner, stars are already aligning - expect a universe of projects, guides, and docs to explore soon. Have a look at who joined our galaxy already:
+### `@astrolicious/i18n`
 
-    ### `@astrolicious/i18n`
+[`@astrolicious/i18n`](https://github.com/astrolicious/i18n) by [Florian](https://github.com/florian-lefebvre) is an i18n integration for Astro with server and client utilities, type safety and translations built-in. It solves some long standing issues regarding translating paths in different locales, including static and dynamic ones! Check the [docs](https://astro-i18n.netlify.app/) to learn more!
 
-    [`@astrolicious/i18n`](https://github.com/astrolicious/i18n) by [Florian](https://github.com/florian-lefebvre) is an i18n integration for Astro with server and client utilities, type safety and translations built-in. It solves some long standing issues regarding translating paths in different locales, including static and dynamic ones! Check the [docs](https://astro-i18n.netlify.app/) to learn more!
+### `astro-tips.dev`
+[astro-tips.dev](https://astro-tips.dev) is an initiative to centralize useful Astro resources that do not fit the official docs but almost! This is just getting started but many maintainers have contributed or are interested in it already!
 
-    ### `astro-tips.dev`
-    [astro-tips.dev](https://astro-tips.dev) is an initiative to centralize useful Astro resources that do not fit the official docs but almost! This is just getting started but many maintainers have contributed or are interested in it already!
+## How Can You Be a Part of This Stellar Journey? ✨
 
-    ## How Can You Be a Part of This Stellar Journey? ✨
+Your support is our rocket fuel! Follow us across the social cosmos. Share our vision with the community and let your voice be heard. We're all ears for feedback, ideas, and dreams. Together, let's make Astrolicious not just successful, but **legendary**.
 
-    Your support is our rocket fuel! Follow us across the social cosmos. Share our vision with the community and let your voice be heard. We're all ears for feedback, ideas, and dreams. Together, let's make Astrolicious not just successful, but **legendary**.
-
-    Stay tuned for more updates by following us on [X](https://twitter.com/astroliciousdev) and on [GitHub](https://github.com/astrolicious). Together, we're not just reaching for the stars — we're shaping the future of the Astro ecosystem. 🚀💫
-
-    ---
-
-    <CreativeCommon title="Hello World, from Astrolicious" titleUrl="https://astrolicious.dev/blog/hello-world" author="Alexander Niebuhr" authorUrl="https://github.com/alexanderniebuhr" />
```

Comment:
I think we should keep having the `CC BY-NC 4.0` opt-in, with an component. But I want to understand why you removed it, can you elaborate?

Reply:
I don't like having it as a component tbh. I think it should appear on any blog post right? So it can leave in the astro file directly

Reply:
IIRC to aligned completely with the license, it would be required to have the **title, url, and author** in the license text. I think that's easier to add with a component though. Additionally if not all content is licensed under CC, and only chosen once, I don't know if it would be confusing in the footer. _Anyways, we can always add the component back later, if we need it._

e.g.
> [Blog Post ](https://astrolicious.dev/)© 2023 by [Alexander ](https://github.com/alexanderniebuhr)is licensed under CC BY-NC 4.0